frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -172,3 +172,33 @@ Feature: Create Application from git form
                   | https://github.com/sclorg/django-ex.git        | django-ex-git-app  | django-ex-git         |
                   | https://github.com/spring-projects/spring-boot | spring-bottom-app  | openshift-quickstarts |
                   | https://github.com/sclorg/nodejs-ex.git        | nodejs-ex-git-app  | nodejs-ex-git         |
+
+
+        @regression @manual
+        Scenario Outline: Dotnet Builder image detection for git url "<git_url>": A-06-TC13
+            Given user is at Import from git page
+             When user enters Git Repo url as "<git_url>"
+              And warning message "Unable to detect the Builder Image" displays in Builder section
+              And user clicks on "Show advanced Git options" link
+              And user clears Context dir field
+              And user enters Context dir as "<dir_name>"
+             Then git url gets Validated
+              And user is able to see "Builder Image(s) detected" message in Builder section
+              And .NET builder image card tile is highlighted with * mark
```

Comment:
Since you are using Examples, lets not bound .NET to the gherkin script, how about using <runtime>?
```suggestion
              And <runtime> builder image card tile is highlighted with * mark
```

Reply:
There is only one example right?, why do we need parameter for this step

Reply:
Usually the scenario Outline Keyword will be used to run the same (generic) scenario multiple times with various values and to avoid repetitions, but here it is only one example and it is bound to dotnet, so that is where I get confused. Either it has to be a generic scenario or specific to dotnet with inline values, as you said it is only one example.